site/docs/pages/index.mdx

```diff
@@ -50,17 +55,14 @@ yarn add @coinbase/onchainkit
 pnpm add @coinbase/onchainkit
 ```
 
-```bash [bun]
-bun add @coinbase/onchainkit
-```
-
 :::
 
       </div>
-      <a href="/getting-started" className="justify-self-end">
-        <button type="button" className="h-[42px] px-8 py-2 rounded-3xl border border-white justify-center items-center gap-2.5 inline-flex">
+      <a href="/getting-started" target="_blank" className="font-sans justify-self-end border rounded-xl border-solid border-white px-4 py-3 font-bold">
```

Comment:
`target="_blank"` ensures the linked document will open in a new tab.

Reply:
Why? it's the same site. It should be in the same tab.

Target Blank should only be used for external sites.

Reply:
Makes sense. Then I'll remove for this link to the same site. And I'll keep the below target blank which link to external figma docs